content/en/docs/setup/install/multicluster/primary-remote/index.md

```diff
@@ -69,6 +75,34 @@ When this feature is enabled, `istiod` will attempt to acquire the leadership lo
 [appropriately annotated](#set-the-control-plane-cluster-for-cluster2) remote clusters that will be
 attached to it (`cluster2` in this case).
 
+{{< /tab >}}
+
+{{< tab name="Helm" category-value="helm" >}}
+
+Install Istio as primary in `cluster1` using standard Helm commands.
+
+First, install the `base` chart in `cluster1`:
+
+{{< text bash >}}
+$ helm install istio-base istio/base -n istio-system --set global.externalIstiod=true --kube-context "${CTX_CLUSTER1}"
```

Comment:
FWIW, `externalIstiod` is not used or templated anywhere in the `base` chart that I can see - it does not need to be set for `base` at all.